.eslintrc-ts

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    "@visionappscz/eslint-config-visionapps",
+    "airbnb-typescript",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
I'm looking at https://typescript-eslint.io/getting-started/typed-linting/ and I can not find `recommended-requiring-type-checking`. All I see is `recommended-type-checked`  and that already includes `@typescript-eslint/recommended`.

@bedrich-schindler could you please take a look at this?